int32 time
string weather
float32 luminance
float32 humidity
float32 temperature
float32 visibility
